docs/examples/model_config_change_globally_baseconfig.py

```diff
@@ -0,0 +1,11 @@
+from pydantic import BaseConfig, BaseModel
+
+BaseConfig.arbitrary_types_allowed = True
```

Comment:
we don't document anywhere doing and I'm still not clear on all its implications.

For now, let's stick to setting the `Config` on models. If we think this is a good idea, it should have an issue to discuss it and its own section in the docs.

Reply:
i think for now, best to remove this example.

Reply:
I removed it. An issue has already been opened here https://github.com/samuelcolvin/pydantic/issues/2194.